controllers/dscinitialization/dscinitialization_controller.go.bk

```diff
@@ -0,0 +1,400 @@
+/*
+Copyright 2023.
```

Comment:
Is that file a backup that should not be checked in?

Reply:
sorry, mistaken, will remove